### DISCUSSION TITLE
Airflow task runner still showing active while task is completed.

### DISCUSSION THREAD

**Discussion by ImSagarbhatt** [apache/airflow#37433] [Q&A]
Hello,

I've recently begun using Apache Airflow version 2.7.3, along with Apache Airflow Providers Celery version 3.5.1 and Celery version 5.3.6.

While my Directed Acyclic Graph (DAG) executed successfully, I encountered an issue where tasks remained active in the Celery Flower UI and in memory even after completion. This persistent activity prevents the worker from picking up new tasks, as it reaches its concurrency limit.

I attempted to address the issue by both downgrading and upgrading the versions of Apache Airflow Providers Celery and Celery, but unfortunately, neither solution resolved the problem.

I've attached an image below illustrating the situation. Could you please assist me in resolving this issue? Thank you.

![image1](https://github.com/apache/airflow/assets/84437086/6bcb02a5-1eea-4bec-bc32-d4ba83b0e87b)

**Comment by potiuk** [comment]
There is far too little information to help you. This is not a known issue, and something possibly on your side, but without you digging deeper, finding where things hang, showing your dags and what you do and more elaboration and possibly comparing your runs with example dags and digging in on your side, this does not seem like anything that was encountered before and there is no advice to be given.

Try to dig deeper and bring more details.
